mediacapture-streams/MediaStream-GUM.https.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>MediaStream getUserMedia test</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#mediastream">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+
+<p class="instructions">Use a test device with color camera(embedded or external).</p>
```

Comment:
Where did the idea  that the camera has to be a color camera come from?
"Camera" should be enough.

Reply:
@alvestrand, thanks for your comment, fixed.